compiler/next/lib/uast/Serial.cpp

```diff
@@ -28,33 +28,45 @@ namespace uast {
 bool Serial::contentsMatchInner(const ASTNode* other) const {
   const Serial* lhs = this;
   const Serial* rhs = (const Serial*) other;
-  return lhs->expressionContentsMatchInner(rhs);
+
```

Comment:
Unless I'm missing something, the checks you make here look the same as what you do in:

Local::contentsMatchInner

Any way we could factor the common code out?

Reply:
That's something that I've been thinking about. I'll probably factor something out into a `BlockLike` or `ConditionalBlock` when I add `On` statements.